evalml/tests/automl_tests/parallel_tests/test_dask_engine.py

```diff
@@ -21,32 +21,12 @@
 )
 
 
-@pytest.fixture(scope="module")
-def cluster():
-    dask_cluster = LocalCluster(
-        n_workers=1, threads_per_worker=1, dashboard_address=None
-    )
-    yield dask_cluster
-    dask_cluster.close()
-
-
-def test_init(cluster):
-    with Client(cluster) as client:
-        engine = DaskEngine(client=client)
-        assert engine.client == client
-
-        with pytest.raises(
-            TypeError, match="Expected dask.distributed.Client, received"
-        ):
-            DaskEngine(client="Client")
-
-
-def test_submit_training_job_single(X_y_binary_cls, cluster):
+def test_submit_training_job_single(X_y_binary_cls):
     """Test that training a single pipeline using the parallel engine produces the
     same results as simply running the train_pipeline function."""
     X, y = X_y_binary_cls
-    with Client(cluster) as client:
-        engine = DaskEngine(client=client)
+
+    with DaskEngine() as engine:
```

Comment:
New use of the context manager for DaskEngine to ensure the resource allocated are deallocated before the next test is run.  This also prevents the pesky "Port # 8787 is occupied" style warnings.  It improves test runtime too.